Formula/firefoxpwa.rb

```diff
@@ -0,0 +1,83 @@
+class Firefoxpwa < Formula
+  desc "Tool to install, manage and use Progressive Web Apps in Mozilla Firefox"
+  homepage "https://github.com/filips123/FirefoxPWA"
+  url "https://github.com/filips123/FirefoxPWA/archive/refs/tags/v0.5.0.tar.gz"
+  sha256 "40004c7a268436173813024672ea23599e387766973d36d0ee0b81793a20b169"
+  license "MPL-2.0"
+  head "https://github.com/filips123/FirefoxPWA.git"
+
+  depends_on "rust" => :build
+
+  def install
+    cd "native"
+
+    # Set the correct version in the source files
+    inreplace "Cargo.toml",
+              "version = \"0.0.0\"",
+              "version = \"#{version}\""
+    inreplace "userchrome/profile/chrome/pwa/chrome.jsm",
+              "DISTRIBUTION_VERSION = '0.0.0'",
+              "DISTRIBUTION_VERSION = '#{version}'"
+
+    # Set the path in the manifest to the Homebrew libexec directory
+    # This is needed so the program respects Homebrew's installation hierarchy
+    # We can use macOS manifest for Linux because the only difference is a default path
+    inreplace "manifests/macos.json",
+              "/usr/local/libexec/firefoxpwa-connector",
+              "#{libexec}/firefoxpwa-connector"
```

Comment:
This should be submitted to be configurable upstream. New formula should not require edits.

Reply:
They there's more rust projects and none of them need patching like this to display the version.

Reply:
Well, that's because I didn't want to change version in the source repository every time I release a new version, so I just embedded `0.0.0`, and set version in GitHub Actions before building the program. However, source code download from GitHub release still contains unmodified source, so I also needed to set the version here.

If edits should not be in formula, maybe I could add some script to upstream that changes the version and called it from formula? But in the end, it will still do basically the same edits (except probably with `sed`)...

Reply:
> If edits should not be in formula, maybe I could add some script to upstream that changes the version and called it from formula? But in the end, it will still do basically the same edits (except probably with sed)

This seems reasonable, in the end it's mostly a question of who needs to maintain it.

Reply:
Ok, I will update upstream with that script soon.

Reply:
I added script to upstream in 0.5.1 and updated formula to use it.